static-site/src/components/ListResources/IndividualResource.jsx

```diff
@@ -33,14 +33,22 @@ export const ResourceLink = styled.a`
   font-size: ${resourceFontSize}px;
   font-family: monospace;
   white-space: pre; /* don't collapse back-to-back spaces */
-  color: ${(props) => props.$hovered ? LINK_HOVER_COLOR : LINK_COLOR} !important;
+  color: ${LINK_COLOR} !important;
   text-decoration: none !important;
+
+  &:hover {
+    color: ${LINK_HOVER_COLOR} !important;
+  }
 `;
 
-function Name({displayName, $hovered, href, topOfColumn}) {
+function Name({displayName, href, topOfColumn}) {
+  const [hovered, setHovered] = useState(false);
+
   return (
-    <ResourceLink href={href} target="_blank" rel="noreferrer" $hovered={$hovered}>
-      {'• '}{($hovered||topOfColumn) ? displayName.hovered : displayName.default}
+    <ResourceLink href={href} target="_blank" rel="noreferrer"
+      onMouseOver={() => setHovered(true)}
+      onMouseOut={() => setHovered(false)}>
+      {'• '}{(hovered||topOfColumn) ? displayName.hovered : displayName.default}
```

Comment:
_non-blocking_

This display logic could also be done in CSS, removing the need entirely for `hovered`, `setHovered`, and `onMouseOver` / `onMouseOut`.

Reply:
Implemented in bfbf4da116563cdc83913db5ff2295dc53ef16b8.

I have a slight preference for the original React implementation since it keeps the parent+child relationship between component and content. I can keep this change though if others think it's better.

Thoughts @jameshadfield @genehack @tsibley?

Reply:
Continued in https://github.com/nextstrain/nextstrain.org/pull/890#discussion_r1628576162